pkg/virtctl/create/vm/vm.go

```diff
@@ -208,8 +229,21 @@ func NewCommand(clientConfig clientcmd.ClientConfig) *cobra.Command {
 		snapshot, params.Supported(dataVolumeSourceSnapshot{}),
 	))
 
+	cmd.Flags().StringVar(&c.user, UserFlag, c.user, "Specify the user in the cloud-init user data that is added to the VM.")
+	cmd.Flags().BoolVar(&c.genPassword, GenPasswordFlag, c.genPassword, "Specify if a password should be generated in the cloud-init user data that is added to the VM.")
+	cmd.Flags().StringVar(&c.passwordFile, PasswordFileFlag, c.passwordFile, "Specify a file to read the password from for the cloud-init user data that is added to the VM.")
+	cmd.Flags().StringSliceVar(&c.sshKeys, SSHKeyFlag, c.sshKeys, "Specify one or more SSH authorized keys in the cloud-init user data that is added to the VM.")
+	cmd.Flags().BoolVar(&c.gaManageSSH, GAManageSSHFlag, c.gaManageSSH, "Specify if the qemu-guest-agent should be able to manage SSH in the cloud-init user data that is added to the VM.")
```

Comment:
I don't think this is useful for now unless we dump secrets and wire up the actual passing of keys through the agent. I'd drop this for now tbh.

Reply:
It is useful in combination with `virtctl credentials add-ssh-key`, which already makes use of the agent to sync authorized keys.

Reply:
Ah, can you add that context to the help text?

Reply:
Done